tests/installation/user_settings.pm

```diff
@@ -28,7 +29,7 @@ sub run {
     }
 
     send_key 'alt-f';    # Select full name text field
-    wait_screen_change { type_string $realname };
+    wait_screen_change { type_string_slow $realname };
```

Comment:
Please only with reference to bug, e.g. a comment.

However, as discussed I think we should invest the effort to do the three steps of:

* detect issue (with the needle)
* delete the wrong content
* type again (but slower to be sure)

or a loop:

```
do {
type
} until (check_correct_username)
```